dist/user-management-auth.js

```diff
@@ -121,17 +169,10 @@
       return ehaUserManagementAuthService.getCurrentUser()
         .then(function(user) {
           if (user && !user.isAdmin() && !user.hasRole(roles)) {
```

Comment:
Is `isAdmin()` still a relevant concept? An admin user should have all roles, shouldn't she?